First draft of full Greenbone CE for Kali Linux documentation (#437)
greenbone/docs: src/22.4/kali/installation.md

```{warning} System upgrades may result in changes to the existing kernel, libraries, and software that could interfere with existing functionality. Before doing an upgrade you should make a complete backup of all critical files on your system including the contents of your PostgreSQL database.
```
```{code-block}
:caption: Full package list update and system upgrade
sudo apt update && sudo apt upgrade
```

### 2. Install Greenbone Community Edition

Greenbone Community Edition and required dependencies can be installed with a single command:

```bash
sudo apt install gvm -y
```
Note that this is the same as executing the command:
```bash
sudo apt install openvas -y
```

### 3. Run The Automatic Configuration Script

After installing the required packages, a configuration script must be run to complete the installation. To run the automatic configuration execute the command:

```{note}

> During `gvm-setup` you must record the default password created for the GVM admin user.

Reviewer: ```suggestion
During `gvm-setup` you must record the default password created for the admin user.
```

```

```bash
sudo gvm-setup
```

### 4. Verify The Installation

The Kali Linux native installation includes a script to verify the installed services. This script can be run using the command:

```bash
sudo gvm-check-setup
```

If your installation has been successfully configured, you will see the following message at the end of the verification out:

```
It seems like your GVM-22.5.0 installation is OK.
```

> #### Starting And Stopping Greenbone CE

Reviewer: ```suggestion
#### Starting And Stopping Greenbone Community Edition
```

The following commands can be used to start and stop Greenbone Community Edition and all its required services:

```{code-block}
:caption: Start all services
sudo gvm-start
```

```{code-block}
:caption: Stop all services
sudo gvm-stop
```

### 5. Log Into The Greenbone Web Interface

Once the installation is complete you can log into the {term}`GSA` web interface by visiting [https://127.0.0.1:9392](https://127.0.0.1:9392) in your browser and providing the default admin credentials from step 3.

### 6. Verify The Feed Status

> Before starting the first scan, Greenbone needs to parse the vulnerability feeds and store them into the `gvmd` PostgreSQL database. This process is initialized during the setup stage, but typically takes anywhere from a few minutes to several hours to complete, depending on your system resources.

Reviewer: ```suggestion
Before starting the first scan, the vulnerability feeds need to be parsed and stored into the `gvmd` PostgreSQL database, otherwise, it will not be able to initialize or complete scans without errors.
This process is initialized during the setup stage, but typically takes anywhere from a few minutes to several hours to complete, depending on your system resources.
```

> The feed status can be checked by going to the `Feed Status` page from the `Configuration` section in the top menu bar.

Author: The grammar of this sentence is confusing.

> The feed status can be checked by going to the `Feed Status` page from the `Configuration` section in the top menu bar.

Reviewer: I've tried to rephrase it. Maybe it's better now?

> The feed status can be checked by going to the `Feed Status` page from the `Configuration` section in the top menu bar.

Reviewer: I've added a new comment to replace this one.
